pages/resources/articles/index.page.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from "react";
```

Comment:
I expect one single generic resources page that accepts the resource type as parameters os it will work for all cases.